src/test/java/seedu/task/logic/LogicManagerTest.java

```diff
@@ -394,6 +394,31 @@ public void execute_select_jumpsToCorrectTask() throws Exception {
     }
 
     @Test
+    public void execute_MarkInvalidArgsFormat_errorMessageShown() throws Exception {
```

Comment:
these two methods are tricky. Please review @TiankaiChen .

Because u called methods that you changed in your Delete branch previously, I am not too sure which versions u intended to use.